plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -449,6 +451,47 @@ void getenv_bool(const char *var, bool *value)
 	pr_info("param: %s:%s\n", var, *value ? "Y" : "N");
 }
 
+void getenv_size_t(const char *var, size_t *value)
+{
+	char *value_str = getenv(var);
+	char *endp = value_str;
+	int sh = 0;
+	size_t size;
+
+	pr_info("Value str: %s\n", value_str);
+
+	if (value_str) {
+		size = (size_t)strtoul(value_str, &endp, 0);
+		if (errno || value_str == endp) {
+			pr_err("Ignoring invalid value for %s=%s, expecting a positive integer\n", var, value_str);
+			return;
+		}
+		switch (*endp) {
+		case 'k':
```

Comment:
should it be the uppercase K?

Reply:
https://en.wikipedia.org/wiki/Units_of_information#Systematic_multiples

> In the past, uppercase K has been used instead of lowercase k to indicate 1024 instead of 1000. However, this usage was never consistently applied.

Reply:
Should be accept both, lowercase and uppercase K to indicate 1024 then?

Reply:
Base CRIU accepts 'K' but not 'k' for its size-parsing config options. The standard is 'k' for 1000. I'm willing to add both.